### PR TITLE
Add mod-right on dropdown

### PR DESCRIPTION
**Pull request by GermainBergeron**
image

### CODE REVIEW

File: docs/_components/controls/12_dropdown-modifiers.html [hunk 1]
```diff
@@ -27,3 +27,27 @@
         </li>
     </ul>
 </div>
+
+<div class="dropdown open right">
```
Comment (JPLachance) [inline docs/_components/controls/12_dropdown-modifiers.html]: What are classes "open right"?

Comment (GermainBergeron) [inline docs/_components/controls/12_dropdown-modifiers.html]: open: opens the dropdown
right: makes the dropdown float to the right

I created neither of them

**Comment by JPLachance** [comment]
:shipit: !

**Comment by wfortin** [comment]
:sheep: it

**Pull request merged**